frontend/.env

```diff
@@ -6,6 +6,7 @@ ESLINT_NO_DEV_ERRORS=true
 
 REACT_APP_ENDPOINT="http://localhost:80/v1"
 REACT_APP_PROJECT=""
+REACT_APP_DOMAIN="http://localhost:3000"
```

Comment:
`domainName` in `frontend/src/utils/config.js` is not an environment variable. The environment variable would be undefined.
I think you shouldn't have removed the definition of `REACT_APP_DOMAIN` because it makes the build process less transparent, less clear for new people. The other environment variables are defined in `./frontend/.env` and therefore `REACT_APP_DOMAIN_NAME` should be there as well.

Reply:
I agree. Maybe at some point we should also create a .env.defaults or so.